apps/frontend-manage/.env.production

```
@@ -11,6 +11,7 @@
 APP_ORIGIN_CONTROL=https://control.klicker.uzh.ch
 APP_ORIGIN_LTI=https://lti.klicker.uzh.ch
 APP_ORIGIN_AUTH=https://auth.klicker.uzh.ch
+APP_ORIGIN_CHAT=https://chat.klicker.uzh.ch
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_ORIGIN_CHAT key should go before the APP_ORIGIN_CONTROL key

(UnorderedKey)